cypher/cypher-docs/src/docs/dev/ql/functions/index.asciidoc

```diff
@@ -74,16 +74,16 @@ These functions take multiple values as arguments, and calculate and return an a
 [options="header"]
 |===
 |Function                                       | Description
-|<<functions-avg,avg()>>                        | Returns the average of a numeric column.
-|<<functions-collect,collect()>>                | Returns a list containing all collected values.
-|<<functions-count,count()>>                    | Returns the number of rows.
-|<<functions-max,max()>>                        | Returns the highest value in a numeric column.
-|<<functions-min,min()>>                        | Returns the lowest value in a numeric column.
+|<<functions-avg,avg()>>                        | Returns the average of a set of numeric values.
```

Comment:
We need to be careful not to imply that these functions take lists and return a single value. They take column (or variable) expressions and use that to aggregate many records down to a single record (based on grouping expressions). The changes you have made here, with the use of the word 'set' seem to imply a real function, not an aggregating expression.